### PR TITLE
Add admin docs.

### PR DESCRIPTION
**Pull request by Smarker**

### CODE REVIEW

File: project-fortis-pipeline/docs/user/admin.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Admin Settings
+
+Project Fortis comes with an admin page with which you can edit site configurations and also provide details on how events should be monitored and processed. Through the admin site, you will be able to configure site settings, blacklist terms, whitelist terms (watchlist), streams (twitter, facebook, etc.), monitored places, and trusted sources.
+
+Once you deployed a site with `deploy to azure`, you may view the admin site by navigating to: http://localhost:8888/#/site/{YOUR_SITE_NAME}/admin
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:5]: If the site is deployed to Azure, it won't be accessible via localhost. This should be the URL provided by the FORTIS_ADMIN_INTERFACE_URL_TAG described in the deployment section. https://github.com/CatalystCode/project-fortis-mono#setting-up-a-new-azure-deployment

File: project-fortis-pipeline/docs/user/admin.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Site Settings
+
+Manage service tokens and set up how the site should be displayed.
+
+| Value               | Description          |
+| ------------------- | ------------- |
+| Site Name           | Used as part of the admin url.  |
+| Site Title          | Displayed on the navigation bar.|
+| Bounding Box        | By default follows `Who's on First (WOF)`. The format of the WOF bounding box is: `maxY,minX,minY,maxX`. `Divipola` is also supported.                           |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:15]: The bounding box is always required to be `maxY,minX,minY,maxX`.

File: project-fortis-pipeline/docs/user/admin.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+| Header Logo Banner  | Image path of banner.  |
+| Supported Languages | A comma delimited string with languages formatted as language codes like `en` or `ar`. The admin site can translate to any of the supported languages.
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:17]: Should link to the full list of supported languages or to the API call that can be used to fetch the supported languages. E.g. http://docs.microsofttranslator.com/text-translate.html#!/default/get_GetLanguagesForTranslate

File: project-fortis-pipeline/docs/user/admin.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+| Default Language    | The admin site will be displayed in this language. |
+| Feature Service     | The feature service connection string formatted as `postgres://frontend:{your_password_here}@127.0.0.1/features`. More documentation on the feature service [here](https://github.com/CatalystCode/featureService).  |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:19]: The feature service database connection string is not configurable. Reading the implementation https://github.com/CatalystCode/project-fortis-mono/blob/7ef5d50850c0e49bb569d12b45dc30967c62c3f7/project-fortis-interfaces/src/components/Admin/AdminSettings.js#L180-L184 informs us that this setting maps to the feature service namespace so it should be `wof` (for whole-world places based on open street maps) or `divipola` (for detailed Colombia places based on the Colombian government data-set).

File: project-fortis-pipeline/docs/user/admin.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+Still needs to be implemented.
+
+## Trusted Sources
+
+Manage trusted sources like `twitter`, `facebook`, etc.
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Pipeline Key        | Key associated with a pipeline. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:66]: What specifically should this be? Given an example please, e.g. for a Twitter pipeline.

File: project-fortis-pipeline/docs/user/admin.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+| External Source Id  | For `facebook`, this is the `facebook page id`. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:67]: What is it for other sources?

File: project-fortis-pipeline/docs/user/admin.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+| Category            | Category monitored in the pipeline for a particular source type. Categories are originally defined in the `watchlist` tab.      |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:68]: What does the category affect?

File: project-fortis-pipeline/docs/user/admin.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+| Name                | Friendly name for trusted source that you can specify as an alias to the external source id. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:69]: Mention that this is used in the UI to display the source.

File: project-fortis-pipeline/docs/user/admin.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+| Source Type         | One of `Twitter`, `Facebook`, `Reddit`, etc. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:70]: Where can a user get the full list?

File: project-fortis-pipeline/docs/user/admin.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+Manage trusted sources like `twitter`, `facebook`, etc.
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Pipeline Key        | Key associated with a pipeline. |
+| External Source Id  | For `facebook`, this is the `facebook page id`. |
+| Category            | Category monitored in the pipeline for a particular source type. Categories are originally defined in the `watchlist` tab.      |
+| Name                | Friendly name for trusted source that you can specify as an alias to the external source id. |
+| Source Type         | One of `Twitter`, `Facebook`, `Reddit`, etc. | 
+| Rank                | An `Integer` value. Specifies the rank of a particular trusted source. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:71]: What does the rank change?

File: project-fortis-pipeline/docs/user/admin.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+| Rank                | An `Integer` value. Specifies the rank of a particular trusted source. |  
+
+## Blacklisted Terms
+
+Manage keywords to blacklist in the pipeline.
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Id                  | Autopopulated `guid` of blacklisted term list. |
+| Blacklisted Terms   | Enter a grouping of keywords for a blacklist as an array of strings `["Trump", "Obama", "election"]`. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:80]: Please given an example of what the blacklist would do in this case.

File: project-fortis-pipeline/docs/user/admin.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+| Pipeline Key        | Key associated with a pipeline. |
+| External Source Id  | For `facebook`, this is the `facebook page id`. |
+| Category            | Category monitored in the pipeline for a particular source type. Categories are originally defined in the `watchlist` tab.      |
+| Name                | Friendly name for trusted source that you can specify as an alias to the external source id. |
+| Source Type         | One of `Twitter`, `Facebook`, `Reddit`, etc. | 
+| Rank                | An `Integer` value. Specifies the rank of a particular trusted source. |  
+
+## Blacklisted Terms
+
+Manage keywords to blacklist in the pipeline.
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:75]: What is the blacklist used for?

File: project-fortis-pipeline/docs/user/admin.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+| Id                  | Autopopulated `guid` of blacklisted term list. |
+| Blacklisted Terms   | Enter a grouping of keywords for a blacklist as an array of strings `["Trump", "Obama", "election"]`. |
+
+## Streams
+
+| Column Name         | Description   |
+| ------------------- | ------------- |
+| Status              | Values are either `enabled` or `disabled`. If a stream is disabled, no events will be collected on that stream in the pipeline. |
+| Pipeline Key        | Key associated with a pipeline. |
+| Params              | Parameters used in different trusted sources like `twitter`, `facebook` etc. For example, for a `twitter` stream, parameters you might have would be `consumer key`, `consumer secret`, `access token key`, `access token secret`. These are formatted as key value pairs. |
```
Comment (c-w) [inline project-fortis-pipeline/docs/user/admin.md:88]: What are these values for other stream types? Where can the user find them?

**Review by c-w — CHANGES_REQUESTED** [review]

**Comment by c-w** [comment]
Great start. I think this could still be made clearer for our users.

Could you please also add a link to these docs in the top-level read-me's deployment guidelines?

**Comment by Smarker** [comment]
Thanks for the detailed review! Yes, I'll also add the link to the top-level.

**Comment by Smarker** [comment]
Closing in favor of a different PR linked to my forked repo as I accidentally messed up this particular pr and did not fork the repo originally.

**Pull request closed without merging**